internal/service/sharing_tf/model.go

```diff
@@ -3693,7 +3693,7 @@ func (c ShareInfo) ApplySchemaCustomizations(attrs map[string]tfschema.Attribute
 	attrs["created_at"] = attrs["created_at"].SetComputed()
 	attrs["created_by"] = attrs["created_by"].SetComputed()
 	attrs["name"] = attrs["name"].SetOptional()
-	attrs["object"] = attrs["object"].SetOptional()
+	attrs["object"] = attrs["object"].SetComputed()
```

Comment:
This needs to remain optional. (universe PR is pending)

```suggestion
	attrs["object"] = attrs["object"].SetOptional()
```